core/src/main/java/org/frankframework/jdbc/DirectQuerySender.java

```diff
@@ -104,14 +101,14 @@ protected void closeConnectionForSendMessage(Connection connection, PipeLineSess
 	@Override
 	// implements IBlockEnabledSender.sendMessage()
 	public SenderResult sendMessage(Connection blockHandle, Message message, PipeLineSession session) throws SenderException, TimeoutException {
-		return new SenderResult(sendMessageOnConnection(blockHandle, message, session, null).getResult());
+		return new SenderResult(sendMessageOnConnection(blockHandle, message, session).getResult());
 	}
 
-	protected PipeRunResult sendMessageOnConnection(Connection connection, Message message, PipeLineSession session, IForwardTarget next) throws SenderException, TimeoutException {
+	protected PipeRunResult sendMessageOnConnection(Connection connection, Message message, PipeLineSession session) throws SenderException, TimeoutException {
```

Comment:
Is deze method nog wel nodig als het aanroepen hiervan het enige is wat `sendMessage` hierboven doet?

Reply:
Ik kijk er even naar!

Reply:
Omdat de method ook nog een override had was het iets meer werk dan alleen maar "inline method" maar ook niet heel veel.